### PR TITLE
Change GetFee method

### PR DESCRIPTION
**Pull request by KyoungBaeLee**
add parameter (amount) to GetFee method

### CODE REVIEW

File: SDK_dotnet/Clients/GluwaClient.cs [hunk 1]
```diff
@@ -380,7 +378,7 @@ private string getGluwacoinTransactionSignature(ECurrency currency, string amoun
 
             if (currency.IsGluwacoinSideChainCurrency())
             {
-                if(currency == ECurrency.sNGNG)
+                if (currency == ECurrency.sNGNG)
```
Comment (nickp8) [inline SDK_dotnet/Clients/GluwaClient.cs:381]: Are these 2 blocks identical?

**Review by nickp8 — APPROVED** [review]

File: SDK_dotnet/Clients/GluwaClient.cs [hunk 2]
```diff
@@ -304,7 +302,7 @@ public GluwaClient(Environment env)
             var result = new Result<bool, ErrorResponse>();
             var requestUri = $"{mEnv.BaseUrl}/v1/Transactions";
 
-            Result<FeeResponse, ErrorResponse> getFee = await getFeeAsync(currency);
+            Result<FeeResponse, ErrorResponse> getFee = await getFeeAsync(currency, amount);
```
Comment (ghost) [inline SDK_dotnet/Clients/GluwaClient.cs:305]: Will this work for all currencies? Only sUSDCG has the variable fee that requires amount to be provided

Comment (nickp8) [reply SDK_dotnet/Clients/GluwaClient.cs:305]: Yes, Fee service will handle it.

Comment (nickp8) [reply SDK_dotnet/Clients/GluwaClient.cs:305]: Although it would be cleaner to not send an amount when you don't need to @KyoungBaeLee

**Pull request merged**